components/atomic/loading/LoadingCircle/LoadingCircle.styles.ts

```diff
@@ -0,0 +1,34 @@
+import styled, { keyframes } from "styled-components";
+import { pxToRem } from "theme/mixins/functions";
+
+export const Skeleton = styled.div`
+  height: 80%;
+  width: 80%;
```

Comment:
Let's set the height here to 100%, and remove the width value. (Width 100% is implied since it's a div). The skeleton should fill the container that it's put in.